pallets/runtime/develop/src/runtime.rs

```diff
@@ -410,7 +410,7 @@ impl pallet_staking::Trait for Runtime {
 }
 
 parameter_types! {
-    pub const MotionDuration: BlockNumber = 0;
+    pub const MotionDuration: BlockNumber = 1;
```

Comment:
That is going to change the behavior, Isn't it? Could you add the release notes and the tag for that.

Reply:
Also could you elaborate on what made this necessary here?

Reply:
Sure. Sorry for the lack of comment. It was changed because `close` doesn't work when `MotionDuration` is 0.